static/src/javascripts/lib/raven.ts

```diff
@@ -52,39 +61,28 @@ const sentryOptions = {
 		return resp;
 	},
 
-	shouldSendCallback(data) {
-		const { isDev } = config.get('page');
-		const isIgnored =
-			typeof data.tags.ignored !== 'undefined' && data.tags.ignored;
-		const { enableSentryReporting } = config.get('switches');
-		const isSentinelLoggingEvent =
-			data?.tags?.tag === 'commercial-sentinel';
+	shouldSendCallback(data: { tags: { ignored?: unknown } }) {
+		const isIgnored = !!data.tags.ignored;
 
-		// isInSample is always true if the tag is commercial-sentinel.
-		// Otherwise, sample at a very small rate.
-		const isInSample = isSentinelLoggingEvent
-			? true
-			: Math.random() < 0.008;
+		// Sample at a very small rate.
+		const isInSample = Math.random() < 0.008;
 
 		if (isDev && !isIgnored) {
-			// Some environments don't support or don't always expose the console Object
-			if (window.console && window.console.warn) {
-				window.console.warn('Raven captured event.', data);
-			}
+			console.warn('Raven captured event.', data);
 		}
 
 		return (
-			enableSentryReporting &&
+			!!enableSentryReporting &&
 			isInSample &&
 			!isIgnored &&
-			!adblockBeingUsed &&
-			(!isDev || isSentinelLoggingEvent)
+			!adblockBeingUsed
 		);
 	},
 };
 
-adblockInUse.then((isUse) => {
+void adblockInUse.then((isUse: boolean) => {
 	adblockBeingUsed = isUse;
 });
 
+// eslint-disable-next-line import/no-default-export -- Allow this default export
```

Comment:
Leaving this one for a future refactor as I want to keep the surface area of this PR as small as possible.